src/services/localstorage.js

```diff
@@ -0,0 +1,10 @@
+function getUser() {
+    const data = localStorage.getItem("linkr");
+    return data ? JSON.parse(data) : null;
+};
+
+function setUser(data) {
+    return localStorage.setItem("linkr", JSON.stringify(data));
+};
+
+export {getUser, setUser};
```

Comment:
**Suggestion:** it makes more sense that this file is inside a folder called utils